examples/assets_dynamic_partitions/README.md

```diff
@@ -0,0 +1,26 @@
+# Dynamic Partitions Example
+
+This example demonstrates how to use dynamically partitioned assets, through a pipeline that processes data about releases of software project – in this case, the Dagster project itself.
+
+It stores the raw data for each release in the local filesystem, and it stores the release metadata in DuckDB.
+
+Most of the assets in the pipeline are partitioned by release. The final asset, `release_summaries`, is not partitioned because it aggregates over all the releases.
+
+A sensor polls the Github API for new releases. When it finds one, it adds it to the set of partitions and runs the pipeline on it - i.e. requests a materialization of the partition corresponding to that release in each of the assets.
+
+
+## Getting started
+
+First, install the project as a Python package. By using the --editable flag, pip will install your Python package in ["editable mode"](https://pip.pypa.io/en/latest/topics/local-project-installs/#editable-installs) so that as you develop, local code changes will automatically apply.
+
+```bash
+pip install -e ".[dev]"
+```
+
+Then, start the Dagster web server:
+
+```bash
+dagster dev
```

Comment:
I would document the required env vars in the readme to be consistent with other quick starts

Reply:
great point